### PR TITLE
Review pass on delegation and voting items. 

### PR DESCRIPTION
**Pull request by LongForWisdom**
Staging Links:
[MKR Holders Guide to Delegation](https://longforwisdom.gitbook.io/new-collection/delegation/for-mkr-holders/mkr-holder-guide)
[Practical Guide to Voting](https://longforwisdom.gitbook.io/new-collection/governance/voting-in-makerdao/practical-guide-voting)

### CODE REVIEW

File: delegation/mkr-holder-guide.md [hunk 1]
```diff
@@ -51,16 +48,16 @@ The table below shows a selection of incentives that delegates can have. It is n
...
 
 * Does this delegate hold MKR? Is it locked for the duration of the delegate contract lifespan?
-* Is this delegate paid by the protocol or an external actor? If the latter, who and what is their objective?
+* Is this delegate paid by the protocol or an external actor? If the latter, who, and what is their motivation?
 * Is this delegate reliant on the Maker Protocol for their own financial stability?
 * Does this delegate have a reputation on the line? Do they care about their reputation?
-* Does this delegate enjoy being a delegate? Have they reliably enjoyed it in the past?
+* Does this delegate enjoy being a delegate? Have they consistently enjoyed it in the past?
 * Does this delegate believe the Maker Protocol is a public good?
 * Does this delegate display strong morals? Are they a trustworthy person?
```
Comment (TheExistence) [inline delegation/mkr-holder-guide.md:63]: Maybe ask us if the delegate is or was a community member before becoming a delegate?

File: delegation/mkr-holder-guide.md [hunk 2]
```diff
@@ -2,40 +2,37 @@
...
 
 ## Trust
 
-Trust has taken on a bad name in the crypto-space. Frequently this is for a good reason. Nevertheless, trust plays a key role in life and human interaction. Trust comes into play heavily in delegation.
+Trust has taken on a bad name in the crypto-space. Frequently this is for a good reason. Nevertheless, trust plays a key role in life and human interaction. Trust comes into play heavily when delegating.
 
-**Delegation is an expression of trust from the MKR Holder to the delegate**
+**Delegation is an expression of the MKR Holder's trust in the delegate**
 
 The fundamental trade-off made by MKR Holders when delegating is to benefit from the increased efficiency of governance, at the cost of trust and centralization. These costs are unavoidable, though they can be minimized by delegating responsibly.
```
[the remark this answers is not in the document]
Comment (LongForWisdom) [reply delegation/mkr-holder-guide.md:23]: Reworded slightly, ended up somewhere in the middel

File: delegation/mkr-holder-guide.md [hunk 3]
```diff
@@ -2,40 +2,37 @@
...
 
-Ultimately the Maker Protocol (and the DAO) needs to have efficient enough governance to operate. If Maker cannot operate, then the MKR token will lose value. Likewise, if delegates collude and attack the Maker Protocol, then the MKR token will lose value.
+Ultimately the Maker Protocol (and the DAO) needs to have efficient enough governance to operate. If Maker cannot operate effectively, then the MKR token will lose value. Likewise, if delegates collude and attack the Maker Protocol, then the MKR token will lose value.
```
[the remark this answers is not in the document]
Comment (LongForWisdom) [reply delegation/mkr-holder-guide.md:25]: Reworded

File: delegation/mkr-holder-guide.md (hunk 1 above)
Comment (LongForWisdom) [reply delegation/mkr-holder-guide.md:63]: I don't think it's a good idea to make that change, like, people being in the community previously isn't a reason to favour or disfavour a delegate. It's more that them being in the community maybe have given them an opportunity to display strong morals, or look favourably on the other questions. Like, it's more important what qualities the delegate has, versus what groups they have previously been a part of.

**Pull request merged**